rocket.chat/content.md

```diff
@@ -0,0 +1,30 @@
+# Rocket.Chat
+
+Rocket.Chat is a Web Chat Server, developed in JavaScript, using the Meteor fullstack framework.
+
+It is a great solution for communities and companies wanting to privately host their own chat service or for developers looking forward to build and evolve their own chat platforms.
+
+%%LOGO%%
+
+# How to use this image
+
+First, start an instance of mongo:
+
+    docker run --name db -d mongo mongod --smallfiles
```

Comment:
Shouldn't need `mongod`, the mongo image just works when you pass in flags :smile:.